app/javascript/containers/ContinueButtonContainer.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { numAnswered, numUnanswered } from '../selectors/answersSelector';
+
+class ContinueButtonContainer extends React.Component {
+    continueOrStart() {
+        if (this.props.numAnswered === 0) {
+            return "Start"
+        } else if (this.props.numUnanswered === 0) {
```

Comment:
Hmm it seems like even buildings that have been fully answered say "Continue" instead of "Review and Submit", but when you click the "Continue" button you get taken to `/review`. How are you calculating `numUnanswered`?